### DISCUSSION TITLE
[Feature] Change album owner

### DISCUSSION THREAD

**Discussion by SimonMellerin** [immich-app/immich#14976] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Today you can 

- create an ablum
- share it
- give right to other user to edit the album

But you can't transfer ownership. 

Could be nice to be able to do it.

Use case : I have freshly installed immich and transferred all our old familly albums with Immich CLI (which is super nice !) but now, I am owner of all these albums. I would like to transfer ownership for some of these albums to my partner.

Thanks a lot for your app which is really goooood ! 

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#13334

**Comment by Marr0w59** [comment]
I would also like to see this.  I also would like to assign more than one owner to a album.  i.e. when sharing to partner have the ability to make them a co-owner of the album.
